src/test/java/steps/Php_Registration.java

```diff
@@ -0,0 +1,94 @@
+        for (int i = 0; i < fieldValues.size(); i++) {
+            hooks.driver.findElement(By.name("firstname")).sendKeys(fieldValues.get(0).get("First Name"));
+            hooks.driver.findElement(By.name("lastname")).sendKeys(fieldValues.get(0).get("Last Name"));
+            hooks.driver.findElement(By.name("phone")).sendKeys(fieldValues.get(0).get("Mobile Number"));
+            //hooks.driver.findElement(By.name("email")).sendKeys(fieldValues.get(0).get("Email"));
+            hooks.driver.findElement(By.name("password")).sendKeys(fieldValues.get(0).get("Password"));
+            hooks.driver.findElement(By.name("confirmpassword")).sendKeys(fieldValues.get(0).get("Confirm Password"));
+            System.out.println("All Data is entered");
+            currentThread().sleep(10000);
+        }
+        Faker faker = new Faker();
+        String email = faker.firstName();
+        hooks.driver.findElement(By.name("email")).sendKeys(email + "@gmail.com");
+
+    }
+
+    @Given("I enter invalid details to below fields")
+    public void invalid(io.cucumber.datatable.DataTable dataTable) throws InterruptedException {
+        List<Map<String, String>> fieldValues = dataTable.asMaps(String.class, String.class);
+        for (int i = 0; i < fieldValues.size(); i++) {
+            hooks.driver.findElement(By.name("firstname")).sendKeys(fieldValues.get(0).get("First Name"));
+            hooks.driver.findElement(By.name("lastname")).sendKeys(fieldValues.get(0).get("Last Name"));
+            hooks.driver.findElement(By.name("phone")).sendKeys(fieldValues.get(0).get("Mobile Number"));
+            hooks.driver.findElement(By.name("email")).sendKeys(fieldValues.get(0).get("Email"));
+            hooks.driver.findElement(By.name("password")).sendKeys(fieldValues.get(0).get("Password"));
+            hooks.driver.findElement(By.name("confirmpassword")).sendKeys(fieldValues.get(0).get("Confirm Password"));
+            System.out.println("All Data is entered");
+            currentThread().sleep(10000);
+        }
+    }
+
+    @When("I click Sign UP button")
+    public void i_click_Sign_UP_button() throws InterruptedException {
+        WebElement element = hooks.driver.findElement(By.xpath("//button[contains(text(),'Sign Up')]"));
+        element.sendKeys(Keys.ENTER);
+        System.out.println("Clicked on Sign up button");
+        currentThread().sleep(10000);
+    }
+
+    @Then("new account must be created")
+    public void new_account_must_be_created() {
+        String url = hooks.driver.getCurrentUrl();
+        System.out.println(url);
+        assertEquals("https://www.phptravels.net/account/", url);
+        System.out.println("New Account Created Successfully ");
+    }
+
+    @Then("User must be navigated to newly created account page")
+    public void user_must_be_navigated_to_newly_created_account_page() {
+        String userName = hooks.driver.findElement(By.xpath("//h3[text()='Hi, Test4 User4']")).getText();
```

Comment:
Test4 might be not generic. Can you think about how we can make this generic for any other user names?